lib/install/proto/installer.proto

```diff
@@ -36,6 +36,9 @@ service Agent {
 
     // Shutdown requests that the installer service shuts down gracefully
     rpc Shutdown(ShutdownRequest) returns (google.protobuf.Empty);
+
+    // GenerateDebugReport requests that the installer generates the debug report
```

Comment:
Nit: Reads a bit weird, how about "requests the installer to generate a debug report".